packages/tools/client-debugger/client-debugger-view/src/components/TelemetryView.tsx

```diff
@@ -99,6 +98,143 @@ export function TelemetryView(): React.ReactElement {
 		};
 	}, [messageRelay, setTelemetryEvents]);
 
+	return (
+		<Stack>
+			<StackItem>
+				<_ListLengthSelection
+					currentLimit={maxEventsToDisplay}
+					onChangeSelection={(key): void => setMaxEventsToDisplay(key)}
+				/>
+			</StackItem>
+			<StackItem>
+				<FilteredTelemetryView telemetryEvents={telemetryEvents} />
+			</StackItem>
+		</Stack>
+	);
+}
+
+/**
+ * {@link _ListLengthSelectionProps} input props.
+ */
+interface _ListLengthSelectionProps {
```

Comment:
I don't think these need to be prefixed with "_". In other places we did that to differentiate components in the same module that would otherwise have the same name, but we generally don't use "_" for private members.

Reply:
Markdown strikes again 😄. There's supposed to be a `_` between each pair of quotes.